client/components/vertical-menu/items/social-item.jsx

```diff
@@ -0,0 +1,64 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import {
+	get,
+	identity
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SocialLogo from 'components/social-logo';
+
+const services = translate => ( {
+	facebook: { icon: 'facebook', label: translate( 'Facebook feed' ) },
+	google: { icon: 'google-plus-alt', label: translate( 'Google search' ) },
+	linkedin: { icon: 'linkedin', label: translate( 'LinkedIn share' ) },
+	twitter: { icon: 'twitter', label: translate( 'Twitter card' ) },
+	wordpress: { icon: 'wordpress', label: translate( 'WordPress.com reader' ) }
```

Comment:
We're going to have to figure something out here, because I see lots of people doing this. But running the translations this way means these strings are not translatable via the community translator. I don't suggest you change this in this PR @dmsnell, but am pointing it out as a common example for @deBhal @akirk and @hewsut to think about addressing in the translator.